packages/clay-time-picker/src/index.tsx

```diff
@@ -51,8 +51,8 @@ interface IProps extends React.HTMLAttributes<HTMLDivElement> {
 		hours: string;
 		minutes: string;
 		clearButton: string;
```

Comment:
I noticed this didn't follow the "language map" pattern, so I updated this in the following commit